python/cudf/setup.py

```diff
@@ -97,7 +146,7 @@
     package_data=dict.fromkeys(
         find_packages(include=["cudf._lib*"]), ["*.pxd"],
```

Comment:
Do we want to add the relevant Protobuf extensions here to make sure those files are included too?

Reply:
The `.proto` file(s) aren't needed at runtime, only when the `build_ext` command is invoked to build Cython extensions. Given that, they shouldn't be included in the installed package, right?

Reply:
Agreed if they are not needed at runtime, we don't need them here.